packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -254,6 +258,29 @@ export const MafsGraph = (props: MafsGraphProps) => {
                             </Mafs>
                         </View>
                     </View>
+                    {state.type === "point" &&
+                        state.showKeyboardInteractionInvitation && (
+                            <View
+                                style={{
+                                    textAlign: "center",
+                                    backgroundColor: "white",
+                                    border: "1px solid #21242C52",
+                                    padding: "16px 0",
+                                    boxShadow: "0px 8px 8px 0px #21242C14",
+
+                                    // This translates the box to the center of the
+                                    // graph Then backs it off by half of its
+                                    // overall height so it's perfectly centered
+                                    top: "50%",
+                                    transform: "translateY(-50%)",
+                                }}
+                            >
```

Comment:
Does the `<LabelMedium>` element need to be wrapped by this `<View>` element, or can the styling here just be applied to the label itself?

Reply:
Yes! We do need the View. It helps with layout. Here's without it:

<img width="503" alt="image" src="https://github.com/user-attachments/assets/52d8c75b-a1f8-4486-b549-c2c6db44e4c9">

Reply:
I think that's because the `<View>` component has `flexbox` automatically applied to it. I would think that adding flexbox properties to the `<LabelMedium>` would accomplish the desired effect. Just looking for ways to reduce `<div>` tags, not a blocker.